docs/reference/build/README.md

```diff
@@ -66,4 +66,4 @@ Apache License Version 2.0
 ## FAQ
 
 Q: Why is it named brodocs?
-A: This project was born out of a collaboration with my brother to create a suitable docs app for his purposes. It was a fun name for the the two of us to use as actual brothers.
\ No newline at end of file
+A: This project was born out of a collaboration with my brother to create a suitable docs app for his purposes. It was a fun name for the two of us to use as actual brothers.
```

Comment:
This file is generated by `kubebuilder docs`, so fixes made to it here will be obliterated by a future update to the API. I believe the original is [here](https://github.com/Birdrock/brodocs/blob/master/README.md), if you would like to fix at the source.